src/adapter/input/post_router.py

```
@@ -20,16 +20,16 @@ def get_post_application_service():
     try:
-        return service.create_post(post)
+        return service.create_post(post_create_request)
     except HTTPException as e:
         raise e
     except Exception as e:
         raise HTTPException(
             status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
-            detail=str(e)
-        )
\ No newline at end of file
+            detail=str(e))
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)